frontend/src/elements/components/ShareData.js

```diff
@@ -0,0 +1,282 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Copy from 'copy-to-clipboard';
+
+import Elements from '../components/Elements';
+import styles from '../theme/ShareData.scss';
+import clipboardSvg from '../../assets/graph_clipboard.svg';
+
+// Get Elements singleton
+Elements();
+
+class EditForm extends React.Component {
+  handleOnSubmit = e => {
+    e.preventDefault();
+    this.props.onSubmit(this.refs);
+  };
+
+  render() {
+    const { onCancel, initialValues = {} } = this.props;
+
+    return (
+      <form className={styles.form} onSubmit={this.handleOnSubmit}>
+        <div className={styles.formName}>
+          <label htmlFor="name">Name</label>
+          <input
+            ref="name"
+            type="text"
+            name="name"
+            id="name"
+            defaultValue={initialValues.name}
+            autoComplete="off"
+          />
+        </div>
+        <button
+          className="button is-small"
+          style={{ position: 'absolute', bottom: 15, left: 15 }}
+          onClick={onCancel}
+        >
+          Go Back
+        </button>
+        <button
+          type="submit"
+          className={`button is-small ${styles.buttonGreen}`}
+          style={{ position: 'absolute', bottom: 15, right: 15 }}
+        >
+          Save
+        </button>
+      </form>
+    );
+  }
+}
+
+const Delete = ({ onDelete, onCancel }) => (
+  <React.Fragment>
+    <p>Are you sure you want to delete this item? This action cannot be rolled back.</p>
+    <div style={{ marginTop: 20 }}>
+      <button onClick={onCancel} className="button">
+        Go back
+      </button>
+      &nbsp;
+      <button onClick={onDelete} className="button is-danger">
+        I'm sure
+      </button>
+    </div>
+  </React.Fragment>
+);
+
+class SharedData extends React.Component {
+  state = {
+    shared: {}
+  };
+
+  componentDidMount() {
+    window.cg.Subject.getSharedData(this.props.sharedDataToken)
```

Comment:
are we actually using this endpoint in the UI?
this is for 3rd parties to use IMO

Reply:
Ouh, I see. I supposed that, but either way, created an specific component to show which data is shared like this:

![image](https://user-images.githubusercontent.com/1077188/42829392-fb415558-89bf-11e8-8ad0-9a0933405707.png)

Reply:
@sindelio @Jakedavies can you chime in here? I think we're counting how many times the URL has been used, are we not? In that case I'd rather have this removed from here and we can get the data from a different endpoint: 
`/api/subject/access-data`

Reply:
I will remove this for now. In next iterations we can decide how is better to allow subjects to share information, since for example they should be able to rectify and also allow/disallow which data they are sharing with kind of "settings" for each share.

Reply:
Have you removed it after all?